schema/sql/api.sql

```diff
@@ -31,9 +31,23 @@ grant usage on sequence "user_id" to "api-user-create";
 grant "api-user-create" to "api";
 grant "api-anon" to "api-user-create";
 
+create role "api-user-manage";
+comment on role "api-user-manage" is
+    $$Intended for use with user management systems$$;
+grant usage on sequence "user_id" to "api-user-manage";
+grant select,insert,update on table
+    public."passwd",
+    public."ssh_public_key",
+    public."openpgp_public_key"
+to "api-user-manage";
+grant "api-user-manage" to "api";
+grant "api-anon" to "api-user-manage";
+grant "api-user-create" to "api-user-manage";
```

Comment:
FYI this is making api-user-manage a superset of api-user-create permissions. Is this intentional?

Reply:
This was marked resolved without answering the question.

Likely you *don't* want this grant.

Reply:
It is intentional, yes. This application needs to be able to create users.